MeasureAuthoringTool/mat-drupal
# Contact us form

## Description
Disabled `search` and `contact` modules. Enabled `ban`, `webform`, `webform_ui`, and `inline_form_errors` modules. Made use of default contact webform, with a few tweaks. Fixed required marker styling for webform elements. Fixed inline error message styling.

## To Test
1. composer install
2. compile pattern lab
3. import config, clear cache
4. Head to `/form/contact` (this will change in the future) and check that the form looks and functions as expected.
